service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/CaseManagementOrderCaseLinkNotificationHandler.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.hmcts.reform.fpl.handlers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.Representative;
+import uk.gov.hmcts.reform.fpl.model.event.EventData;
+import uk.gov.hmcts.reform.fpl.service.InboxLookupService;
+import uk.gov.hmcts.reform.fpl.service.RepresentativeService;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.CaseManagementOrderEmailContentProvider;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.commons.lang.StringUtils.isNotBlank;
+import static uk.gov.hmcts.reform.fpl.NotifyTemplates.CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE;
+import static uk.gov.hmcts.reform.fpl.enums.RepresentativeServingPreferences.DIGITAL_SERVICE;
+
+@Component
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class CaseManagementOrderCaseLinkNotificationHandler {
+    private final ObjectMapper objectMapper;
+    private final InboxLookupService inboxLookupService;
+    private final NotificationService notificationService;
+    private final RepresentativeService representativeService;
+    private final LocalAuthorityNameLookupConfiguration localAuthorityNameLookupConfiguration;
+    private final CaseManagementOrderEmailContentProvider caseManagementOrderEmailContentProvider;
+
+    public void sendNotifications(final EventData eventData) {
+        sendToLocalAuthority(eventData);
+        sendToRepresentatives(eventData);
+    }
+
+    private void sendToLocalAuthority(final EventData eventData) {
+        final String localAuthorityName = localAuthorityNameLookupConfiguration.getLocalAuthorityName(
+            eventData.getLocalAuthorityCode());
+
+        final Map<String, Object> localAuthorityNotificationParameters = caseManagementOrderEmailContentProvider
+            .buildCMOIssuedCaseLinkNotificationParameters(eventData.getCaseDetails(), localAuthorityName);
+
+        final String email = inboxLookupService.getNotificationRecipientEmail(eventData.getCaseDetails(),
+            eventData.getLocalAuthorityCode());
+
+        notificationService.sendEmail(CMO_ORDER_ISSUED_CASE_LINK_NOTIFICATION_TEMPLATE, email,
+            localAuthorityNotificationParameters, eventData.getReference());
+    }
+
+    private void sendToRepresentatives(final EventData eventData) {
```

Comment:
Fairly similar to the `sendToRepresentatives` on CaseManagementOrderDocumentLinkNotificationHander. Is there room for abstraction maybe?

Reply:
as discussed